src/styles/blocks/calendar.scss

```
@@ -0,0 +1,74 @@
+.calendar {
+  height: 100vh;
+
+  &__title {
+    display: flex;
+    justify-content: center;
+
+    padding-top: 25px;
+
+    font-size: 55px;
+    color: black;
+    cursor: pointer;
+
+    &:hover {
+      text-shadow: 3px 3px pink;
+    }
+  }
+
+  &__content {
+    display: flex;
+    flex-wrap: wrap;
+    gap: $day-gap;
+
+    max-width: $calendar-width;
+
+    padding: 25px $calendar-padding;
+
+    margin: auto;
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-grow: 0;
+    flex-shrink: 0;
+    flex-basis: $day-square-size;
+
+    height: $day-square-size;
+
+    text-align: center;
+    border: 1px solid black;
+
+    background-color: $day-color;
+
+    cursor: pointer;
+    transition: background-color 0.5s, transform 0.5s;
+
+    &:hover {
+      box-shadow: 5px 5px gray;
+      background-color: pink;
+      transform: translate(-5px, -20px);
+    }
+
+    @for $index from 1 through 31 {
+      &:nth-child(#{$index})::before {
+        content: "#{$index}";
+        @extend %calendar-font;
+      }
+    }
+  }
+
+  @each $day, $index in $days {
+    &--start-day--#{$day} &__day:nth-child(1) {
+      margin-left: getFirstDayShift($index);
```

```suggestion
      margin-left: getFirstDay($index);
```

This is not working, because you mispelled the name.

---

Yep